### DISCUSSION TITLE
Grouping Logitech Media Server Players

### DISCUSSION THREAD

**Discussion by sotatech** [music-assistant/support#796] [Show and tell]
Firstly, thank you for this excellent addition to Home Assistant! I tried creating a HA Media Player group and adding two of my Logitech Radios to it. The two devices are in adjacent rooms, so it quickly became obvious that they were drifiting out of sync. Not unexpected as I believe Music Assistant does not support perfect sync (yet?).
My solution was to use an LMS plugin which allows you to create player groups: https://forums.slimdevices.com/showthread.php?108558-ANNOUNCE-Group-Players&highlight=group 
The grouped players appear as a single media player device in Music Assistant and can be added the same as any other device. As LMS is managing the sync, the players never drift. Obviously, you cannot control the individual members of the group with Music Assistant, but otherwise it works fine for me.

**Comment by marcelveldt** [comment]
Sync is indeed not yet supported and is something that is currently in the process of being added. You can expect it to arrive somewhere within the next few weeks.

**Comment by sotatech** [comment]
Thanks, Marcel. I look forward to trying this out!

**Comment by OzGav** [comment]
Old V1 info
